providers/cloudflare/config/config.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+package config
+
+import (
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/plugin"
+	"go.mondoo.com/cnquery/v11/providers/cloudflare/connection"
+	"go.mondoo.com/cnquery/v11/providers/cloudflare/provider"
+)
+
+var Config = plugin.Provider{
+	Name:            "cloudflare",
+	ID:              "go.mondoo.com/cnquery/v11/providers/cloudflare",
+	Version:         "11.0.0",
+	ConnectionTypes: []string{provider.DefaultConnectionType},
+	Connectors: []plugin.Connector{
```

Comment:
you should add a flag for providing the cloudflare token to make it consistent with other providers. We always have the option to use a flag or env var for specifying tokens